packages/main/src/DayPicker.ts

```diff
@@ -51,7 +51,7 @@ import DayPickerTemplate from "./generated/templates/DayPickerTemplate.lit.js";
 // Styles
 import dayPickerCSS from "./generated/themes/DayPicker.css.js";
 
-const isBetween = (x: number, num1: number, num2: number) => x > Math.min(num1, num2) && x < Math.max(num1, num2);
```

Comment:
Originally @hinzzx and thought it might be better to reduce the *minimal* overload caused by calling the math.min and max functions but after discussing it we changed it back for better readability.